lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,35 @@
 import React, { Component } from 'react';
 import Card from './Card';
+import PropTypes from 'prop-types'
 
 const Cards = props => {
   return (
     <div className="cards-container">
-      {/* Using the cards prop, map over the list creating a 
-          new Card component for each passing the card as the only prop*/}
+  
+          {props.cards.map((post, i) => {
+            return (
+            <Card 
+            key={i}
+            post={post} 
+            headline={post.headline} 
+            img={post.img} 
```

Comment:
Because you are sending all the post info here, you only need the key and the post={post}